collect_app/src/main/java/org/odk/collect/android/activities/AboutActivity.java

```diff
@@ -83,52 +85,57 @@ private void initToolbar() {
 
     @Override
     public void onClick(int position) {
-        switch (position) {
-            case 0:
-                websiteTabHelper.openUri(this, websiteUri);
-                break;
-            case 1:
-                forumTabHelper.openUri(this, forumUri);
-                break;
-            case 2:
-                Intent shareIntent = new Intent(Intent.ACTION_SEND);
-                shareIntent.setType("text/plain");
-                shareIntent.putExtra(Intent.EXTRA_TEXT,
-                        getString(R.string.tell_your_friends_msg) + " " + GOOGLE_PLAY_URL
-                                + getPackageName());
-                startActivity(Intent.createChooser(shareIntent,
-                        getString(R.string.tell_your_friends)));
-                break;
-            case 3:
-                boolean intentStarted = false;
-                try {
-                    // Open the google play store app if present
-                    Intent intent = new Intent(Intent.ACTION_VIEW,
-                            Uri.parse("market://details?id=" + getPackageName()));
-                    List<ResolveInfo> list = getPackageManager().queryIntentActivities(intent, 0);
-                    for (ResolveInfo info : list) {
-                        ActivityInfo activity = info.activityInfo;
-                        if (activity.name.contains("com.google.android")) {
-                            ComponentName name = new ComponentName(
-                                    activity.applicationInfo.packageName,
-                                    activity.name);
-                            intent.setComponent(name);
-                            startActivity(intent);
-                            intentStarted = true;
+        //debounce clicks
+        //actions that do not change the activity should set the flag to false once they finish!
+        if (!handlingClickFlag) {
```

Comment:
please check [Collect.allowClick()](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/application/Collect.java#L358) method. I think it should be used here.

Reply:
I searched a bit for examples on how this method is used and I also noticed (for example in MainMenuActivity when the onClicks are set) that clicks are also call ActivityLogger.logAsAction(Object t, String context, String action)
Should I include this call on my PR? If so, what should I pass as the ”context” parameter?